tests/test_localtime.py

```diff
@@ -853,6 +853,12 @@ class TestCombine:
                 "UTC",
                 datetime.datetime(2020, 1, 1, tzinfo=pytz.UTC),
             ),
+            (
```

Comment:
🐼 Instead of adding an extra parameter, we could add a second parametrize call that uses either `pytz.UTC` or `datetime.timezone.utc` for each of the existing parameters.